package.json

```diff
@@ -11,28 +11,27 @@
   "scripts": {
     "build": "microbundle",
     "format": "prettier --write src/**/*",
-    "test": "env NODE_ICU_DATA=$(node-full-icu-path) jest",
+    "test": "jest",
     "prepublish": "in-publish && yarn build || :"
   },
   "publishConfig": {
     "access": "public"
   },
   "devDependencies": {
-    "@types/jest": "^23.0.0",
-    "eslint": "^4.19.1",
-    "eslint-plugin-prettier": "^2.6.0",
-    "eslint-plugin-shopify": "^22.1.0",
-    "full-icu": "^1.2.1",
+    "@types/jest": "^26.0.23",
+    "eslint": "^7.26.0",
+    "eslint-plugin-prettier": "^3.4.0",
+    "eslint-plugin-shopify": "^35.1.0",
     "in-publish": "^2.0.0",
-    "jest": "^23.1.0",
-    "microbundle": "^0.4.4",
-    "prettier": "^1.13.4",
-    "ts-jest": "^22.4.6",
-    "typescript": "^2.9.1"
+    "jest": "^26.6.3",
+    "microbundle": "^0.13.0",
+    "prettier": "^2.2.1",
+    "ts-jest": "^26.5.6",
+    "typescript": "^4.2.4"
   },
   "jest": {
     "transform": {
-      "^.+\\.ts$": "<rootDir>/node_modules/ts-jest/preprocessor.js"
+      "^.+\\.ts$": "ts-jest"
```

Comment:
`ts-jest` told me to do this in a warning when I would run `yarn test`, so I did it 🤷‍♂️